components/Add/Add.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Add = () => {
+  return 
```

Comment:
return something so you can verify that routing works.

Reply:
Done.